apps/atomy-q/API/app/Http/Controllers/Api/V1/QuoteSubmissionController.php

```
@@ -110,13 +123,17 @@ public function upload(QuoteSubmissionUploadRequest $request): JsonResponse
+        $qs->confidence = $this->shouldUseManualExtractionContinuity() ? null : 85.0;
         $qs->line_items_count = 0;
         $qs->warnings_count = 0;
         $qs->errors_count = 0;
         $qs->save();
 
-        $this->dispatchProcessingJob($qs);
+        if ($this->shouldUseManualExtractionContinuity()) {
+            $this->markExtractionUnavailable($qs);
+        } else {
+            $this->dispatchProcessingJob($qs);
+        }
```

PHPMD (2.15.0)

[error] The method upload uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)